specification/search/resource-manager/Microsoft.Search/stable/2015-08-19/examples/SearchDeleteQueryKey.json

```diff
@@ -0,0 +1,14 @@
+{
+    "parameters": {
+        "searchServiceName": "mysearchservice",
+        "resourceGroupName": "rg1",
+        "key": "Query key for browser-based clients",
```

Comment:
>Query key for browser-based clients [](start = 16, length = 35)

since this is an examples, should we put a fake key here?

Reply:
You found a bug in the example. Query keys are supposed to be deleted by value, not by name.

I'll use the same placeholder syntax here that I've used elsewhere. Looking at examples in other Swagger specs, I don't see any of the other Azure services using fake values. Some reasons I can think of for this include:
 1. Avoiding accidentally putting a real value in the examples.
 2. Avoiding triggering cred-scanning false positives.

Reply:
Make sense. The two points is what I though you were trying to avoid by putting the description of what the value is supposed to be, using a placeholder is a better idea.